From 7747fa30d1d5a99dd00cba1e9da9b24b84de1927 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Tue, 21 May 2024 15:27:20 +0200
Subject: [PATCH 0846/1695] dts: stm32: enable PKA on stm32mp215f-dk
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Sets PKA peripheral status to okay.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 4d6234bdc6f120c06e4fe537ee5e46e10c0fded6 dts: stm32: enable PKA on stm32mp215f-dk]
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Change-Id: I909de8aec1ef49c4e4e7b997a2084d394b1876a1
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/381351
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp215f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp215f-dk.dts b/core/arch/arm/dts/stm32mp215f-dk.dts
index e5f6f47ad..5f0ddc4e9 100644
--- a/core/arch/arm/dts/stm32mp215f-dk.dts
+++ b/core/arch/arm/dts/stm32mp215f-dk.dts
@@ -53,6 +53,10 @@
 	status = "okay";
 };
 
+&pka {
+	status = "okay";
+};
+
 &rcc {
 	st,c1msrd = <2>;
 	st,clk_opp {
-- 
2.47.2

